main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"os"
+
+	"github.com/hudl/zendesk-exporter/config"
+	"github.com/hudl/zendesk-exporter/logging"
+	"github.com/hudl/zendesk-exporter/ticketwriter"
+	"github.com/hudl/zendesk-exporter/zendesk"
+
+	"github.com/adamar/ZeGo/zego"
+	golog "github.com/op/go-logging"
+)
+
+var log = golog.MustGetLogger("main")
+
+func main() {
+	logging.Configure()
+	log.Info("Starting...")
+
+	startTime := os.Args[1]
```

Comment:
Actually, yes, I do. I'm thinking if this line panics, your more helpful message will never get hit.